extensions/interactions/GraphInput/directives/graph-input-rules.service.ts

```diff
@@ -179,14 +197,15 @@ export class GraphInputRulesService {
 
     // Check against every permutation of vectices.
     var numVertices = graph2.vertices.length;
-    var permutation = [];
+    var permutation: number[] | null = [];
```

Comment:
Please add a comment on why this can be null ?

Reply:
done